base/doc/clsguide-historic.tex

```diff
@@ -34,15 +34,15 @@
 
 \title{\LaTeXe~for class and package writers --- historic version}
 
-\author{Copyright \copyright~1995--2006 The \LaTeX\ Project\\
+\author{Copyright \copyright~1995--2023 The \LaTeX\ Project\\
```

Comment:
2006 to 2023 is a big step. Should this happen?

Reply:
yes I think it should